recipes/dexplot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "dexplot" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a9aee9685a2a69f4243684c75077a7cea012973c4cff4ce0a661a5c880973a2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.15
+    - pandas >=0.24
+    - python >=3.6
+    - scipy >=1.0
+    - matplotlib-base >=3.1
+
+test:
+  imports:
+    - dexplot
+  commands:
+    - pip check
```

Comment:
:tada:

Reply:
Heh, i did it!

I used grayskull, and it added `pip check`. It did use just `matplotlib` and actually had it appended to the line above it (probably bug).